pennylane/tape/qnode.py

```diff
@@ -402,18 +402,15 @@ def construct(self, args, kwargs):
             # controlled rotations aren't supported by the parameter-shift rule
             stop_at = set(self.device.operations) - {"CRX", "CRZ", "CRY", "CRot"}
 
-        obs_wires = [wire for m in self.qtape.measurements for wire in m.wires if m.obs is not None]
-
-        obs_on_same_wire = len(obs_wires) != len(set(obs_wires))
+        # pylint: disable=protected-access
+        obs_on_same_wire = len(self.qtape._repeated_observables) > 0

```

Comment:
Very very minor, but 'repeated' makes it sound like the case `qml.PauliY(0), qml.PauliY(1)` (e.g., the _same_ observables, not (potentially different) observables on the same wires.

Is there a better term? All I can think of is 'overlapping', but that is also very handwavey.

Reply:
Thanks, I've updated to `_obs_sharing_wires`